CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ UNRELEASED
 * [ [#1504](https://github.com/digitalfabrik/integreat-cms/issues/1504) ] Keep filters on pagination
 * [ [#1585](https://github.com/digitalfabrik/integreat-cms/issues/1585) ] Hide news after 28 days
 * [ [#1600](https://github.com/digitalfabrik/integreat-cms/issues/1600) ] Improve XLIFF export bulk option description
+* [ [#1599](https://github.com/digitalfabrik/integreat-cms/issues/1599) ] XLIFF only includes draft and public translations
```

Comment:
Maybe `XLIFFs never include translations in review or auto saved` is explaining it better?